MIM-NEN2660/config.js

```diff
@@ -46,10 +46,10 @@ var respecConfig =
   //specStatus: "GN-DEF",             // Definitieve Versie
   //-- specType is verplicht bij alle andere dan BASIS ---------------------------------
   //specType: "NO",                   // Norm
-  specType: "HR",                   // Standaard
```

Comment:
Is dit een bewuste change in deze PR?

Reply:
Ik vermoed dat @PalmJanssen deze onbewust in deze branch heeft gecommit. Verder lijkt het me niet heel erg. Functioneel is er niks veranderd aan die config.